antlr/src/main/antlr/FSH.g4

```diff
@@ -120,6 +143,9 @@ REFERENCE:          'Reference' WS* '(' WS* SEQUENCE WS* ('|' WS* SEQUENCE WS*)*
                  //  ^  NON-WHITESPACE
 CARET_SEQUENCE:     '^' ~[ \t\r\n\f]+;
 
+                 // '/' EXPRESSION '/' FLAGS
+REGEX:              '/' ('\\/' | ~[/\r\n])+ '/' REGEX_FLAGS;
```

Comment:
Did you determine that there's a place to put those flags in the FHIR VS?  It seemed to me like there is not, but I wasn't sure if you researched it further.  If there's not, then I don't think we should support it in the grammar.

Reply:
Since the value field in a filter is just a string, filters could only be included if the stored string was something like `/expression/flags` and not just `expression`. But, that seems like the sort of fact that would be included in the documentation if it were true, so I guess it expects the value to just be `expression`, which unfortunately means no support for flags.

Reply:
Yeah, that's the basic conclusion I came to as well. :-/